### ISSUE TITLE
Forgotten project?

### ISSUE THREAD

**Issue by ccartola** [lujjjh/iLyrics#14]
Had this on my mac and it worked really good. Should take time to make it work in Windows too as well as Mac works.

Congrats! Hoping your work doesn't end now.  Not all songs show lyrics by now.

**Comment by lujjjh** [comment]
Hey @ccartola!

Based on your description, I'm guessing you are building iLyrics from the sciter branch (#8), as the main branch still targets iTunes on Windows and is incompatible with macOS.

I'm rewriting iLyrics in #8 to be cross platform and system wide rather than an add-on for iTunes. But I've encountered two main problems that block this process:

1. There are rendering issues with Sciter on Windows. Sciter is not open-source, so I'll have to either find some workaround, or wait for the author to fix it. Another choice is to switch to another cross-platform GUI solution.
2. iLyrics needs the information of current playing song to show the time-synced lyrics. However, [very few music players](https://github.com/ModernFlyouts-Community/ModernFlyouts/blob/0.9.3/docs/GSMTC-Support-And-Popular-Apps.md) on Windows provides the timeline information. Even Chrome, Edge and Firefox are not included for the moment. This means that the new iLyrics does not support any popular browsers on Windows, which makes the rewrite meaningless.

Recently, I'm trying to fix the second problem. I've contributed to Chromium ([chromium/chromium#e36fce](https://github.com/chromium/chromium/commit/e36fce28b77dd55d54fe902f968de5543e5dfe11)) which should make the new iLyrics compatible with Chrome and Edge.

**Comment by ccartola** [comment]
Hi Jiahao.

I'm running iLyrics in Windows 10 with iLyrics.exe

The problem I see is not all songs show their lyrics as well as Mac does. The reason I see is iLyrics for Mac is an app, you can control a lot of things: wrong lyrics, fonts...

That's what I meant when I wrote that: I know iLyrics is running but I can't control so many things as I did in Mac. 

So my advice is: try to do a program or app as it is, not an apendix of iTunes in Windows that the user can't control.

Is really wonderful what you do and I would like to thank you publicly.

**Comment by lujjjh** [comment]
Thanks for your suggestion. That's what I'm going to do in #8.

> The problem I see is not all songs show their lyrics as well as Mac does. The reason I see is iLyrics for Mac is an app, you can control a lot of things: wrong lyrics, fonts...

To make it clear, I haven't published iLyrics for Mac, so what you describe may be another app with the same name. Maybe you can share that app here. :)

**Comment by ccartola** [comment]
I thought I had an app called as the same as you, but it was LyricsX. You can see if you want how the author did it. It's really good. Had a database really nice, most of them from China.

**Comment by lujjjh** [comment]
Yeah, I'm aware of this app. In fact, this is not entirely due to the lyrics database / providers.

There is currently no way to manually search for and choose lyrics, so iLyrics uses a tougher standard to match the lyrics to prevent incorrect lyrics from being displayed (i.e. false positive). However, this will also prevent correct lyrics of some songs from being matched (i.e. false negative).
